apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -462,10 +462,17 @@ export class PortfolioService {
     });
 
     const holdings: PortfolioDetails['holdings'] = {};
-    const totalInvestmentInBaseCurrency = currentPositions.totalInvestment.plus(
+    const totalInvestmentInBaseCurrency = currentPositions.currentValue.plus(
       cashDetails.balanceInBaseCurrency
     );
-    let filteredValueInBaseCurrency = currentPositions.currentValue;
+
+    const isFilteredByAccount = filters.some(
+      (filter) => filter.type === 'ACCOUNT'
```

Comment:
I'm always using the written out version. Can you adapt for harmonization:

`(filter) => { return filter.type === 'ACCOUNT'; }`